test/types/actions.js

```diff
@@ -0,0 +1,221 @@
+const nullAsAny = /** @type {any} */(null);
```

Comment:
I changed TS to JS + JSDoc for this file, But GitHub doesn't recognize file renaming🤷

Reply:
I think for these files we should keep typescript. They are only internal for tests and not transformed or bundled in any way, so it doesn't affect the output.

Reply:
Yeah, I thought the same thing.
I don't have a strong opinion but we can get almost the same type checking between TS and JSDoc, so "Let's use JSDoc always" is super simple to recognize how to write tests.
And we can execute these type tests also if we want.
That's why I converted to JSDoc for type tests also.

(On the other hand, JSDoc has a limitation, so using TS for type tests can implement more tests.)

Anyway, I'm not against reverting to TS!

Reply:
@dummdidumm I reverted type tests!